README.md

```diff
@@ -23,23 +23,34 @@ cargo install cargo-wipe
 cargo wipe --help
 ```
 
-## target
+## Usage
+
+To find build folders for `language` that can potentially be deleted run

```

Comment:
In order to keep this consistent with the cli help, can you also please add `name = "language"` to https://github.com/mihai-dinculescu/cargo-wipe/blob/master/src/command.rs#L13